python/servo/bootstrap.py

```diff
@@ -384,14 +384,19 @@ def get_linux_distribution():
             base_version = '10.10'
         else:
             raise Exception('unsupported version of %s: %s' % (distro, version))
-
         distro, version = 'Ubuntu', base_version
+    elif distro.lower() == 'ubuntu':
+        if version == '19.04':
+            base_version = '19.04'
+        else:
+            raise Exception('unsupported version of %s: %s' % (distro, version))
```

Comment:
I believe we want to keep supporting earlier versions of Ubuntu. Can we revert this?